src/ape/types/trace.py

```diff
@@ -325,7 +325,7 @@ def get_content_asts(self, location: SourceLocation) -> List[ASTNode]:
         return [
             a
             for a in self.ast.get_nodes_at_line(location)
-            if a.lineno >= location[0] and a.classification == ASTClassification.CONTENT
+            if a.lineno >= location[0] and a.classification != ASTClassification.FUNCTION_DEF
```

Comment:
I think this also works, and maybe be suggested for enums
```suggestion
            if a.lineno >= location[0] and a.classification is ASTClassification.FUNCTION_DEF
```